Darknet/Assets/Scripts/TK_Ani.cs

```diff
@@ -2,13 +2,81 @@
 using System.Collections;
 
 public class TK_Ani : MonoBehaviour {
-	
-	// Update is called once per frame
-	void Update () {
-//	   if (Input.GetKey (KeyCode.D))
-//			animation.CrossFade ("Test-kun_Walk6");
-//	   else
-//			animation.CrossFade ("Test-kun_Idle5");
-		}	
+
+private Animator animator;	
+// Use this for initialization
+void Start(){
+ animator = this.GetComponent<Animator>();
+}
+
+
+
+// Update is called once per frame
+void Update () {
```

Comment:
I don't think your update function is checking conditions correctly. Check out the link that I provided in the Slack. The animation code the poster shows is more complicated; it checks to make sure that a specific key is pressed *and* that you are *not* in a certain state.